docs/getting-started/detections-req.asciidoc

```diff
@@ -0,0 +1,124 @@
+[[detections-permissions-section]]
+= Detections prerequisites and requirements
+
+To use the <<detection-engine-overview, Detections feature>>, you need to
+configure a few things.
+
+IMPORTANT: There are a number of step that are *only* required for *on-premises*
+{stack} deployments. If you are using a cloud deployments, you only need to
+configure <<enable-detections-ui>> and <<access-detections-ui>>.
+
+Additionally, there are some <<adv-list-settings, advanced settings>> used to
+configure {kib} <<detections-ui-exceptions, value list>> upload limits.
+
+You need the https://www.elastic.co/subscriptions[appropriate license] to send
+<<rule-notifications, notifications>> when detection alerts are generated.
+
+[discrete]
+[[detections-on-prem-requirements]]
+== Configure on-premises {stack} deployments
+
+These steps are only required for *on-premises* deployments:
+
+* HTTPS must be configured for communication between
+{kibana-ref}/configuring-tls.html#configuring-tls-kib-es[{es} and {kib}].
+* In the `elasticsearch.yml` configuration file, set the 
+`xpack.security.enabled` setting to `true`. For more information, see 
+{ref}/settings.html[Configuring {es}] and
+{ref}/security-settings.html[Security settings in {es}].
+* In the `kibana.yml` {kibana-ref}/settings.html[configuration file], add the 
+`xpack.encryptedSavedObjects.encryptionKey` setting with any alphanumeric value 
+of at least 32 characters. For example:
++
+`xpack.encryptedSavedObjects.encryptionKey: 'fhjskloppd678ehkdfdlliverpoolfcr'`
+
+IMPORTANT: After changing the `xpack.encryptedSavedObjects.encryptionKey` value
+and restarting {kib}, you must restart all detection rules.
+
+[discrete]
+[[enable-detections-ui]]
+== Enable Detections
+
+To enable the <<detection-engine-overview, Detections feature>>, a user with
+these privileges must visit (click on) the *Detections* page:
+
+* The `manage` cluster privilege
+* {kib} space `All` privileges for the `Security` feature (see
+{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges]).
```

Comment:
We may want to mention adding `Saved Objects Management` here (or for users to initiate/create rules) just in case they're using this user for more than just `Enable Detections`, as these steps will allow them to 'unlock' the detections page, but the user will still see an error when creating a rule.